std_msgs/String options # filter options
---
ObjectPosition[] objects
std_msgs/String[] keys